hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -61,6 +69,15 @@ def lambda_handler(event, context):
 
         perform_label_action(action_message, action_message.action_label)
 
+        ActionEventRecord(
+            content_id=action_message.content_key,
+            time_performed=datetime.datetime.now(),
+            action_label=action_message.action_label.value,
+            # Hack: the label rules model is not something I don't fully understand yet...
+            # rn this just says f-it let's make a list of json blobs we can recover and store it.
+            action_rules=[rule.to_aws_json() for rule in action_message.action_rules],
```

Comment:
Is the eventual goal to save a reference to the immutable and versioned action_rule object?

This tells the rules, but doesn't say why the rules existed. If there were an audit, this would be hard to explain without some normalizing.

Reply:
Why the rule existed would be: __\<User x\> removed condition 'Y' on \<date\>.__

Reply:
yeah, this is a little bit of a mess... 
Storing the whole rule object as is for now to avoid falling down another rabbit hole in this PR. 

Larger issues around recording action for an audit exist beyond the scope of this PR and might be worth a design review. cc: @eriksendc  and #596